src/PlaywrightSharp.Tests/Page/PageOpenerTests.cs

```diff
@@ -1,4 +1,5 @@
 using System.Threading.Tasks;
+using PlaywrightSharp.Helpers;

```

Comment:
We could undo all the extra usings right?

Reply:
done, removed all unused usings in test project